### PR TITLE
PR for Badges

### PR DESCRIPTION
**Pull request by manojasarsa**
Added Badges which includes:

1. Badges on Avatar
2.  Badges on Icons

### CODE REVIEW

File: css/common.css [hunk 1]
```diff
@@ -66,13 +66,3 @@ select {
 h1, h2, h3, h4, h5, h6, p {
```
Comment (msdeshmukh009) [inline css/common.css:66]: It's actually recommended that you should avoid styling elements directly.

**Review by msdeshmukh009 — APPROVED** [review]
All looks good to me.

**Review by Varu98 — APPROVED** [review]
apart from  what Mahesh said, the code and the class convention are so on point, GJ! 👍🏻

File: css/common.css (hunk 1 above)
Comment (manojasarsa) [reply css/common.css:66]: We can define like this in the common css (reset.css) file. Right?

**Pull request merged**